pennylane/tape/qscript.py

```diff
@@ -1143,37 +1143,42 @@ def specs(self):
             dict[str, Union[defaultdict,int]]: dictionaries that contain quantum script specifications
 
         **Example**
+        >>> prep = [qml.BasisState(np.array([1,1]), wires=(0,"a"))]
+        >>> ops = [qml.DoubleExcitation(0.1, wires=[0, 1, 2, 3])]
+        >>> qscript = QuantumScript(ops, [qml.expval(qml.PauliZ(0))], prep)
 
-        >>> ops = [qml.Hadamard(0), qml.RX(0.26, 1), qml.CNOT((1,0)),
-        ...         qml.Rot(1.8, -2.7, 0.2, 0), qml.Hadamard(1), qml.CNOT((0,1))]
-        >>> qscript = QuantumScript(ops, [qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))])
```

Comment:
Why the example needs to be changed?

Reply:
I think I copied the doc-string from the PRD. Reverting the example now 👍🏼

Reply:
Fixed